modules/artifact/src/main/java/org/mule/runtime/module/artifact/api/classloader/MuleArtifactClassLoader.java

```diff
@@ -199,7 +200,7 @@ public URL findResource(String name) {
             // We don't want class loaders in limbo
             synchronized (descriptorMappingLock) {
               if (descriptorMapping.get(matchDescriptor) == null) {
-                URLClassLoader urlClassLoader = new URLClassLoader(new URL[] {url});
+                URLClassLoader urlClassLoader =  new URLClassLoader(new URL[] {url}, getParent(), getCachingURLStreamHandlerFactory());;
```

Comment:
wouldn't the equivalent be to use the system classloader instead of the parent?

Reply:
Was not sure, I could , wanted to be saafe